templates/pharmacies/general_edit.html

```diff
@@ -1,73 +1,78 @@
-<table style="font-size:9pt;" width="500px" CELLSPACING="0" CELLPADDING="3">
-<tr>
-	<td width="150px" VALIGN="MIDDLE" >{xl t='Name'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="40" name="name" value="{$pharmacy->name|escape:'html'}" onKeyDown="PreventIt(event)" />({xl t='Required'|escape:'html'})
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Address'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="40" name="address_line1" value="{$pharmacy->address->line1|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Address'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="40" name="address_line2" value="{$pharmacy->address->line2|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='City, State Zip'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="25" name="city" value="{$pharmacy->address->city|escape:'html'}" onKeyDown="PreventIt(event)" /> , <input type="text" size="2" maxlength="2" name="state" value="{$pharmacy->address->state|escape:'html'}" onKeyDown="PreventIt(event)" /> <input type="text" size="5" name="zip" value="{$pharmacy->address->zip|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Email'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input TYPE="TEXT" NAME="email" SIZE="35" VALUE="{$pharmacy->email|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Phone'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input TYPE="TEXT" NAME="phone" SIZE="12" VALUE="{$pharmacy->get_phone()|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Fax'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input TYPE="TEXT" NAME="fax" SIZE="12" VALUE="{$pharmacy->get_fax()|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Default Method'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<select name="transmit_method">{html_options options=$pharmacy->transmit_method_array selected=$pharmacy->transmit_method}</select>
-	</td>
-</tr>
-<tr><td colspan="2">&nbsp;</td></tr>
-<tr>
-	<td colspan="2">
-	    <div class="btn-group">
-	        <a href="javascript:submit_pharmacy();" class="btn btn-default btn-save" onclick="top.restoreSession()">
-	            {xl t='Save'|escape:'html'}
-	        </a>
-	        <a href="controller.php?practice_settings&pharmacy&action=list" class="btn btn-link btn-cancel" onclick="top.restoreSession()">
-                {xl t='Cancel'|escape:'html'}
-            </a>
-        </div>
-	</td>
-</tr>
-</table>
-<input type="hidden" name="id" value="{$pharmacy->id|escape:'html'}" />
-<input type="hidden" name="process" value="{$PROCESS|escape:'html'}" />
+    <input type="hidden" name="form_id" value="{$pharmacy->id|escape:'html'}" />
+	<div class="form-group">
```

Comment:
I was questioning value of spending development effort of replacing `table` by `div`.  If you were to write completely new code in responsive world for large number of devices, it makes sense to start with div.  But in cases like this, reworking the code manually is a luxury for the sake of purity when merely adding table class to existing code and stripping all td formatting let's bootstrap.js do the same rendering.